### DISCUSSION TITLE
LiquidMenu and PlatformIO

### DISCUSSION THREAD

**Discussion by feoktant** [VasilKalchev/LiquidMenu#78] [Ideas]
Hi, thank you for great lib!

Recently, I started to use PlatformIO, LiquidMenu and LiquidCrystal_I2C. I tried to find how to change `LiquidMenu_config.h` file for building with I2C changes, but without success. I tried to import code [in lib dependencies](https://community.platformio.org/t/cant-modify-library-thats-linked-in-platformio-ini/7523/5), but got bunch of errors about imports <> and "" chars.

Finally, I found how to manage this:
```
lib_ldf_mode = deep
lib_deps =
	marcoschwartz/LiquidCrystal_I2C@^1.1.4
	vase7u/LiquidMenu@^1.6.0
lib_ignore =
    LiquidCrystal
```

Could such example be added into https://github.com/VaSe7u/LiquidMenu/tree/master/examples/I_I2C_menu ?
Is this right approach?

**Comment by VasilKalchev** [comment]
Thanks for the suggestion, @feoktant and sorry for the late reply.

I'm not sure if this is the right approach since I haven't used PlatformIO. Another suggestion, in the discussion you linked, was to [manually place the library inside `lib/`](https://community.platformio.org/t/cant-modify-library-thats-linked-in-platformio-ini/7523/2).

Since this is specific to PlatformIO, I'll just refer people to this post if they experience the same problem, for now.
